Datahub.Web/.env.example

```diff
@@ -22,3 +22,10 @@ AWS_SECRET_ACCESS_KEY=someawssecretaccesskey
 
 # A Google Tag Manager ID for use in analytics, not used at dev time and defaults to ""
 GTM_ID=GTM-XXXXXXX
+
+# Reference URL to point where this is going to be hosted at (used in sitemap.xml/robots.txt generation)
+SELF_REFERENCE_URL = localhost
```

Comment:
This variable is a host, not a URL so should be `SOMETHING_HOST`.... and `SELF_REFERENCE` doesn't explain what it is. I found v. confusing til I looked at how it was used. Something like `ENVIRONMENT_HOST` would be better.

Better still would be to not introduce this var - it's available in the environment's `HttpContext`. https://stackoverflow.com/questions/53577352/get-host-address-in-asp-net-core-mvc

One less thing to setup / configure.

Reply:
Removed reference